default/python/AI/FreeOrionAI.py

```diff
@@ -146,7 +146,7 @@ def prepareForSave():  # pylint: disable=invalid-name
     in the state string so that they can be restored if the game is loaded."""
     empire = fo.getEmpire()
     if empire is None:
-        print "This client has no empire. Doing nothing to prepare for save."
+        info("This client has no empire. Doing nothing to prepare for save.")
```

Comment:
As above, this seems like enough of an unexpected situation that it should be debug or warn rather than merely info.